src/test/java/integration/ClearCookiesTest.java

```diff
@@ -0,0 +1,29 @@
+package integration;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.Cookie;
+
+import java.util.Set;
+
+import static com.codeborne.selenide.Selenide.clearBrowserCookies;
+import static com.codeborne.selenide.WebDriverRunner.getWebDriver;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class ClearCookiesTest extends IntegrationTest {
+  @Before
+  public void addCookiesBeforeTest() {
+    getWebDriver().manage().addCookie(new Cookie("username", "John Doe"));
+    Set<Cookie> cookieSet = getWebDriver().manage().getCookies();
+    assertFalse(cookieSet.isEmpty());
```

Comment:
It looks like here you test selenium webdriver feature but not selenide.
Or change to assumeFalse if you afraid that some implementations of webdriver can't add cookies in some cases